server/apps/main/views.py

```diff
@@ -523,82 +524,113 @@ def moderate_experience(request, uuid):
         model = PublicExperience.objects.get(experience_id=uuid)
         if request.method == "POST":
             # get the data from the model
-            moderated_form = ModerateExperienceForm(request.POST)
-            moderated_form.is_valid()
-
-            # Get the (immutable) experience data
-            unchanged_experience_details = extract_experience_details(model)
```

Comment:
I know that this isn't where the `model` is introduced, but just to raise that it probably should be called `public_experience` or something similar as it's confusing to have such an abstract variable name :D

Reply:
Agreed; I've taken the opportunity to rename it! Thanks 😃